docs-2.0/8.service-tuning/load-balance.md

```diff
@@ -108,12 +115,12 @@
 ```ngql
 nebula> SUBMIT JOB BALANCE DATA REMOVE 192.168.8.100:9779;
```

Comment:
during my test, I was using `BALANCE DATA REMOVE 192.168.8.100:9779;` instead of `SUBMIT JOB xxx`, both of the two work? or?